python/cudf/cudf/tests/test_series.py

```diff
@@ -1348,6 +1348,18 @@ def test_nullable_bool_dtype_series(data, bool_dtype):
     assert_eq(psr, gsr.to_pandas(nullable=True))
 
 
+@pytest.mark.parametrize(
+    "cudf_series", [(cudf.Series([0.25, 0.5, 0.2, -0.05]))]
```

Comment:
Can we cover more cases like inputs containing `np.nan`, `cudf.NA`? Also parameterize `lag` too?